src/deepsparse/transformers/__init__.py

```diff
@@ -41,9 +41,9 @@
 def _check_transformers_install():
     import transformers as _transformers
 
-    if not _transformers.NM_INTEGRATED:
+    if not hasattr(_transformers, "NM_INTEGRATED") or not _transformers.NM_INTEGRATED:
```

Comment:
I guess `not _transformers.NM_INTEGRATED` is a bad check since base won't have this at all.
```suggestion
    if not getattr(_transformers, "NM_INTEGRATED", False):
```

Reply:
Nice call, much more clear